doc/source/user_guide/tutorial_parallelization.rst

```diff
@@ -51,6 +51,18 @@ Another equivalent way to code this loop is to use a comprehension list which ha
 
     %timeit comprehension_loop()
 
+``apply_parallel`` is a function in ``skimage.util`` module that provides mapping function in parallel across an array.
```

Comment:
I think the grammar in "that provides mapping function" is off? How about

```suggestion
:func:`skimage.util.apply_parallel` can map a function in parallel across an array.
```

This seems a lot more concise to me. By using the :func: role we can make Sphinx add a link to the documentation of the function.